### DISCUSSION TITLE
Can't have kg.m^2 but always g.m^2

### DISCUSSION THREAD

**Discussion by s-celles** [hgrecco/pint#2083] [Q&A]
Hi

I'm doing

```python
from pint import UnitRegistry
ureg = UnitRegistry(system="SI")
J = 66.845 * ureg('g*m^2')
J_kg = J.to('kg*m^2')
mo.md(f"J = {J_kg:.3f~#P}")  # Marimo markdown display
```

I think it should display : J = 0.067 kg⋅m² but I'm getting J = 66.845 g·m²

What's wrong here?

Kind regards

**Comment by andrewgsavage** [comment]
`f"J = {J_kg:.3f~P}"`

see https://pint.readthedocs.io/en/stable/user/formatting.html#quantity-modifiers
